src/sentry/grouping/ingest/config.py

```diff
@@ -94,6 +95,11 @@ def update_grouping_config_if_permitted(project: Project) -> None:
             event=audit_log.get_event_id("PROJECT_EDIT"),
             data={**changes, **project.get_audit_log_data()},
         )
+        metrics.incr(
+            "grouping.config_updated",
+            sample_rate=options.get("grouping.config_transition.metrics_sample_rate"),
+            tags={"current_config": current_config, "source": source},
```

Comment:
Do we need to track `source`? What are we trying to measure?

Reply:
Not strictly necessary, but I figured it'd be since to be able to distinguish upgrades done by the script from upgrades which happen organically.